### PR TITLE
Basic QA tools - php-cs-fixer and phpstan

### PR DESCRIPTION
**Pull request by mateuszsip**
To not overthink installation process (composer deps vs phars) I used `jakzal/phpqa:alpine` image and added `php-cs-fixer` and `phpstan` to lint target. These two are usually sufficient.

What do you think?

Related to #8

### CODE REVIEW

**Comment by matigda** [comment]
Looks ok :) IMO just add mess detector check to it and we're good to go :)

File: Makefile [hunk 1]
```diff
@@ -13,3 +13,10 @@ test: ## Run tests
 
 run_local:  ## Run local environment
 	sudo docker-compose up -d
+
+lint: ## Run qa checks
+	docker run -it --rm -v $(PWD):/app -w /app jakzal/phpqa:alpine php-cs-fixer fix --dry-run --diff src
```
Comment (matigda) [inline Makefile]: I suggest @Symfony rules at least.

**Review by matigda — CHANGES_REQUESTED** [review]
Add mess detector and symfony rules in cs-fixer check and we're good to go.

**Comment by mateuszsip** [comment]
PHPMD `naming` and `controversial` rulesets are not included, some of them would be false positives and php-cs-fixer already covered what really matters.

After merging we need to make code pass linters checks.

**Comment by matigda** [comment]
@kejwmen resolve conflicts :)

**Comment by mateuszsip** [comment]
rebased.

**Review by matigda — APPROVED** [review]

**Pull request merged**